src/System.IO.FileSystem/ref/System.IO.FileSystem.cs

```diff
@@ -228,7 +228,7 @@ public partial class FileStream : System.IO.Stream
         public override void WriteByte(byte value) { }
         public virtual void Unlock(long position, long length) { }
     }
-    public abstract partial class FileSystemInfo
+    public abstract partial class FileSystemInfo : System.MarshalByRefObject, System.Runtime.Serialization.ISerializable
```

Comment:
I see ISerializable is on the implementation (in the src folder) so I believe this is correct. @stephentoub